python/cusignal/_spectral.py

```diff
@@ -287,6 +292,19 @@ def _populate_kernel_cache():
             "_cupy_lombscargle"
         )
 
+    else:
+        if str(numba_type) in _numba_kernel_cache:
+            return
+        sig = _numba_lombscargle_signature(numba_type)
+        _numba_kernel_cache[(str(numba_type))] = cuda.jit(sig, fastmath=True)(
+            _numba_lombscargle
+        )
+
+
+def _precompile_kernels(dtype, use_numba=False):
```

Comment:
If this is intended as part of the user-facing API, would it be better to present it without a leading underscore?